library/src/containers/Sidebar/Sidebar.tsx

```diff
@@ -383,7 +383,7 @@ const OperationItem: React.FunctionComponent<OperationItemProps> = ({
     <li>
       <a
         className="flex no-underline text-gray-700 mb-2 hover:text-gray-900"
-        href={`#operation-${kind}-${channelName}`}
+        href={'#' + channelName}
```

Comment:
@AceTheCreator do you recall how will it affect v2 vs v3 compatibility?

@Santhoshmani1 I see in your video that you text with AsyncAPI v2. Can you see how it works with v3 -> https://github.com/asyncapi/spec/blob/master/examples/streetlights-kafka-asyncapi.yml

I'm not sure change is so trivial as in case of v2 operations are part of channels, while in v3 they are not, and there is no pub/sub kinds 🤔 

nevertheless for sure it is not enough just to fix href here. For example when you will check the v3 document that I shared with you, the operation id is `operation-receive-receiveLightMeasurement` where `receive` is `kind`, and `receiveLightMeasurement` is not `channelName` but it is the name of operation. You probably need to extend `OperationItem` component to also have access to operation name, and us it in href 🤔